tasks/telemetry.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Build language telemetry map
+  set_fact:
+    telemetry_library_key:
+      java: specified_java_lib_version
+      js: specified_nodejs_lib_version
+      dotnet: specified_dotnet_lib_version
+      python: specified_python_lib_version
+      ruby: specified_ruby_lib_version
+
+- name: Build APM library installation payloads for telemetry
+  set_fact:
+    telemetry_payload:
+      "{{ telemetry_payload | default([]) |
+          combine({telemetry_library_key[item.split(':')[0]]:
+            item.split(':')[1] | default('default')}) }}"
+  loop: "{{ pre_processed_datadog_apm_libraries }}"
+
+- name: Build telemetry request
+  set_fact:
+    telemetry_request:
+      request_type: 'apm-onboarding-event'
+      api_version: 'v2'
+      application:
+        service_name: "{{ ansible_hostname }}"
+      payload:
+        event_name: 'agent.injection.success'
+        tags: "{{ {'install_time': ansible_date_time.epoch | int,
+                   'install_type': 'ansible'} |
+                   combine(telemetry_payload) }}"
+
+- name: Send installation telemetry to Datadog
```

Comment:
Let's add `when: not ansible_check_mode` for this task.

Reply:
Done